examples/newsreaderExample/src/main/java/io/realm/examples/newsreader/NewsReaderApplication.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.newsreader;
+
+import android.app.Application;
+import android.content.Context;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+import rx.plugins.RxJavaErrorHandler;
+import rx.plugins.RxJavaPlugins;
+import timber.log.Timber;
+
+public class NewsReaderApplication extends Application {
+
+    private static Context context;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        context = this;
+
+        // Logging and error handling from RxJava
+        Timber.plant(new Timber.DebugTree());
```

Comment:
Kind of a nit of mine, but I usually prefer to see the debug tree in the debug build variant and then a release tree in the release variant. Just a nit though, nothing huge. :)